OMS.Web/AdminController.generated.cs

```diff
@@ -427,13 +427,13 @@ public override System.Web.Mvc.ActionResult CreateAdmin(OMS.Core.DTO.Account acc
         [NonAction]
         partial void RoleOverride(T4MVC_System_Web_Mvc_ActionResult callInfo);
 
-        [NonAction]
-        public override System.Web.Mvc.ActionResult Role()
-        {
-            var callInfo = new T4MVC_System_Web_Mvc_ActionResult(Area, Name, ActionNames.Role);
-            RoleOverride(callInfo);
-            return callInfo;
-        }
+      //  [NonAction]
+       // public override System.Web.Mvc.ActionResult Role()
+        //{
+         //   var callInfo = new T4MVC_System_Web_Mvc_ActionResult(Area, Name, ActionNames.Role);
+          //  RoleOverride(callInfo);
+           // return callInfo;
+       // }
```

Comment:
why did you commented out this code? this file is a generated file and you shouldn't touching this file and let the T4MVC generate the code